### PR TITLE
Development review build

### PR DESCRIPTION
**Pull request by Menubrea**
Implemented features as requested in the CA. The build is pretty barebone, and nothing beyond what has been requested has been worked on. Plenty of room for improvements on UX specifically and overall functionality - assuming we are to continue working on this project post-delivery.

Known issues: 
- Nav toggle needs improvements and better spacing.
- Posts under profile.html is not outputing the correct spacing on large screens.
- Code cleanup not finalized.

#Feedback:
- Would like feedback on spacing, particularly vertical spacing for both mobile and desktop.
- Is the overall design theme appropriate?
- Issues with code/bootstrap (better ways to write it).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,68 +8,73 @@
   <script defer src="/node_modules/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
   <link rel="stylesheet" href="dist/css/styles.css">
   <script src="https://kit.fontawesome.com/6b9295215b.js" crossorigin="anonymous"></script>
-  <title>#connect | Home</title>
+  <link rel="stylesheet" href="https://use.typekit.net/bqw8pcm.css">
```
Comment (sulenchy) [inline index.html:11]: To improve the SEO of this page, you can add a metadata for description

File: login.html [hunk 2]
```diff
@@ -1,83 +1,49 @@
...
-  <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
-  <title>Document</title>
-</head>
-<body>
-  <!DOCTYPE html>
-<html lang="en">
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (sulenchy) [inline login.html:6]: You can add metadata for description here too

**Review by sulenchy — APPROVED** [review]
@ClovesPirate I left some comments concerning the description metadata.

**Comment by Menubrea** [comment]
# To reviewer

The overall UI is pretty basic, as the assignment does not require a proper dashboard you'd expect from a social platform. Assuming we are to build upon this project a few things planned for the future for a better UX:
- Dashboard navigation on larger screen sizes/drawer for smaller screens.
- Modal/drawer chat functionality with easy access to friends.
- The ability to explore content and other users based on preference and interests.
- A more robust login page with more information provided to the potential user.

Otherwise it should be feature-complete as outlined in the assignment.

# changes
- Updated head to include a description for better SEO as requested by reviewer.
- Added .gitignore as per peer request
`- I used git in the console to add a .gitignore to my /node_modules using [this](https://sebhastian.com/git-ignore-node_modules/) tutorial, however it still appears in my text-editor, but it is grayed out. Not sure if I did it correctly. Feedback on what to do differently would be greatly appreciated it it is not working as intended.`

**Review by sulenchy — CHANGES_REQUESTED** [review]
The file changes are too much. This is a result of you pushing node_modules to GitHub. Please, do this to get that fixed

```
git rm -r --cached node_modules
git commit -m 'fix: remove node_modules from origin'
git push origin development
```

`git rm -r --cached node_modules` returns pathspec 'node_modules' did not match any files

**Review by sulenchy — CHANGES_REQUESTED** [review]
Please get rid of the node_modules

**Review by sulenchy — APPROVED** [review]
Looks good to me

**Pull request merged**